code/src/components/InputForm.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+const InputForm = ({input, setInput}) => {
+    
+    return (
+        <div className="input-form-div">
```

Comment:
It doesn't seem like you use `className="input-form-div"` in your stylesheet. Do you think it's better if you remove it, if you are not using it?